### ISSUE TITLE
About Symfony version of the dependencies

### ISSUE THREAD

**Issue by mickaelandrieu** [piwik/piwik#18437]
## Summary

I noticed the project is using a deprecated version of Symfony (2.6 while we 5.4/6.0 was published recently).

I was wondering why you don't upgrade the Symfony version of these packages ?

Can we help ?

## Your Environment

I don't use Matomo __yet__

**Comment by sgiehl** [comment]
We have meanwhile updated to a newer version if Symfony.

**Comment by mickaelandrieu** [comment]
Yes, and I'm using the most recent version of Matomo in production 👍
